### PR TITLE
Feature/keyboard

### PR DESCRIPTION
**Pull request by Aniket310101**
Completed Challenge 2: Jiggling Keys.

### CODE REVIEW

File: Challenge4/.idea/STARTER-FILES.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vrknetha) [inline Challenge4/.idea/STARTER-FILES.iml]: add these .idea to git ignore. this should not be part of your source code

File: Challenge4/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+const keys = document.querySelectorAll(".key");
+
+let keyPressed;
+let keyCurrentlyJiggling;
+let numberOfKeys = keys.length;
+
+
+const crossValidateTheKeyPressed = () => {
+  if (keyPressed === keyCurrentlyJiggling) {
+    console.log("true");
```
Comment (vrknetha) [inline Challenge4/index.js]: remove console log

**Review by vrknetha — CHANGES_REQUESTED** [review]

File: Challenge4/index.js (hunk 2 above)
Comment (Aniket310101) [reply Challenge4/index.js]: Resolved.

File: Challenge4/.idea/STARTER-FILES.iml (hunk 1 above)
Comment (Aniket310101) [reply Challenge4/.idea/STARTER-FILES.iml]: Resolved

**Pull request merged**